scenarios/ocd/CreateAKSDeployment/README.md

```diff
@@ -211,14 +205,14 @@ kubectl apply -f azure-vote-start.yml
 Validate that the application is running by either visiting the public ip or the application url. The application url can be found by running the following command:
 
 > [!Note]
-> It often takes 2-3 minutes for the PODs to be created and the site to be reachable via http
+> It often takes 2-3 minutes for the PODs to be created and the site to be reachable via HTTP
 
 ```bash
 runtime="5 minute";
 endtime=$(date -ud "$runtime" +%s);
 while [[ $(date -u +%s) -le $endtime ]]; do
    STATUS=$(kubectl get pods -l app=azure-vote-front -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}'); echo $STATUS;
-   if [ "$STATUS" = 'True' ]; then
+   if [ "$STATUS" == 'True' ]; then
```

Comment:
both `=` and `==` are valid for string comparisons but I agree that `==` feels more natural